python/servo/platform/base.py

```diff
@@ -112,6 +113,13 @@ def install_taplo(self, force: bool) -> bool:
 
         return True
 
+    def install_crown(self, force: bool) -> bool:
+        if subprocess.call(["cargo", "install", "--path", "support/crown"],
+                           stdout=subprocess.PIPE, stderr=subprocess.PIPE) != 0:
+            raise EnvironmentError("Installation of crown failed.")
+
+        return True
+
```

Comment:
There's an issue here which is that updates to crown won't be automatically installed when building.

Reply:
That's why it's always forced. It would be nice if we could be more smart about it, but I do not have any idea how.

Reply:
Perhaps when `crown` starts it could check a value in the workspace Cargo.toml which specifies the minimum version. If it is less than the current version it could emit and error and request running `./mach bootstrap` again. `./mach bootstrap` could always try to update to the latest version of `crown.

Reply:
This could work, but crown is run for every rustc invocation, which could slow down compilation.

Reply:
Maybe storing good rust version in env var should speed things up, but then we can have old data in env var.